bzt/modules/blazemeter.py

```diff
@@ -1452,10 +1451,17 @@ def check(self):
     def post_process(self):
         if not self.detach and self.router and not self.test_ended:
             self.router.stop_test()
+
         if self.results_url:
             if self.browser_open in ('end', 'both'):
                 open_browser(self.results_url)
 
+        if self.router and self.router.master:
+            full = self.router.master.get_full()
+            for session in full.get('sessions', ()):
+                for error in session.get("errors", ()):
+                    raise TaurusException(to_json(error))
```

Comment:
1. NetworkError is unappliable here, 'cause it's not network error. It's something inside BlazeMeter cloud failed, network is fine. I decided to not create new exception class just because of single situation.
2. In practice, there won't be many different messages. I decided to start with something very simple and see if it will be sufficient.